src/api/app/controllers/person/token_controller.rb

```diff
@@ -50,5 +51,17 @@ def set_package
 
       @package = Package.get_by_project_and_name(params[:project], params[:package])
     end
+
+    def validate_operation
+      operation_param = params[:operation]
+      return if operation_param.nil? || Token::OPERATIONS.include?(operation_param.capitalize)
```

Comment:
Is this line changing from the allowed token type `runservice` to `service`? :thinking:

This is already documented [here](https://github.com/openSUSE/open-build-service/blob/221811444256005c46b48af45f7f6313496c8499/src/api/public/apidocs/paths/person_login_token.yaml#L89).

Reply:
Ah, I think I misused https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/token.rb#L27 set of possible values instead. But now I'm puzzled: why the API uses a different set than the webUI?

Reply:
I added a change to handle and accept both webUI and API values keeping the same behavior otherwise.

https://github.com/openSUSE/open-build-service/pull/14979/files#diff-67a22ba8cf70b37462f7ef3e9fe0370283ffae0f2ace759387591b1bbb06324dR57-R62

But it still feels dirty and hacked somehow, so we should rather align the two usages eventually.